chapters/equations.tex

```diff
@@ -244,13 +244,18 @@ \subsubsection{Defining When-Equations by If-Expressions in Equality Equations}\
 The start-values of the introduced \lstinline!Boolean! variables are defined by the taking the start-value of the when-condition, as above where \lstinline!b! is a parameter
 variable.  The start-value of the special functions \lstinline!initial!, \lstinline!terminal!, and \lstinline!sample! is \lstinline!false!.
 
-\subsubsection{Restrictions on Equations within When-Equations}\label{restrictions-on-equations-within-when-equations}
+\subsubsection{Restrictions on When-Equations}\label{restrictions-on-when-equations}
+
+\paragraph*{Restrictions on where a when-equation may occur:}
```

Comment:
Is it really needed to use `\paragraph*` for introducing a list like this?

Then, the paragraph break before the list environment should be removed for optimal layout with LaTeX.